index.html

```diff
@@ -213,5 +213,115 @@ <h1>Skills</h1>
         </div>
       </div>
     </section>
+
+    <section id="contact-form-section">
+      <div class="logoimage">
+        <a href="index.html"
+          ><img
+            class="logoimage"
+            src="images/image_geometry_1(1).svg"
+            alt="My portfolio's logo"
+        /></a>
+      </div>
+      <div>
+        <h1 class="contact name">Get started</h1>
+        <p class="bio form-text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+        <form action="https://formspree.io/f/mgedpadw" method="POST">
+          <div class="form-container">
+            <label for="username"></label>
+            <input
+              type="text"
+              id="username"
+              name="user_name"
+              placeholder="Your name"
+              maxlength="30"
+              required
+            />
+          </div>
+          <div class="form-container">
+            <label for="email"></label>
+            <input
+              type="text"
+              id="email"
+              name="e-mail"
+              placeholder="Your email"
+              required
+            />
```

Comment:
- [ ] Kindly make the type attribute of your email here an [`email`](https://www.w3schools.com/tags/att_input_type_email.asp#:~:text=The%20%3Cinput%20type%3D%22email,add%20the%20%22multiple%22%20attribute.) type. This ensures that the user always enter an email in the input otherwise it will give an error message.